administrator/components/com_content/controllers/article.php

```diff
@@ -154,8 +155,35 @@ public function batch($model = null)
 	 *
 	 * @since	3.1
 	 */
-	protected function postSaveHook(JModelLegacy $model, $validData = array())
+	protected function postSaveHook(JModelLegacy $model, $validData)
 	{
+		JModelLegacy::addIncludePath(JPATH_ADMINISTRATOR . '/components/com_menus/models', 'MenusModel');
+		$itemModel = JModelAdmin::getInstance('Item', 'MenusModel');
+		$itemModel->addTablePath(JPATH_ADMINISTRATOR . '/components/com_menus/tables');
+
+		$componentId = JComponentHelper::getComponent('com_content')->id;
+
+		JLoader::register('ContentHelper', JPATH_ADMINISTRATOR . '/components/com_content/helpers/content.php');
+		$articleId = ContentHelper::getarticleid($validData['title']);
+
+		$vaData['id'] = 0;
```

Comment:
`$vaData` is not valid name for the variable. It should be a meaning name. Like, `$data` or `$itemData`